fuzzer.cpp

```diff
@@ -113,6 +113,16 @@ void Fuzzer::ParseOptions(int argc, char **argv) {
   incremental_coverage = GetBinaryOption("-incremental_coverage", argc, argv, true);
   
   add_all_inputs = GetBinaryOption("-add_all_inputs", argc, argv, false);
+
+  process_name = GetOption("-process_name", argc, argv);

```

Comment:
I wonder if attach mode could be fully implemented in `TinyInstInstrumentation` class, without requiring any changes to the `Fuzzer` class.